mediapipe/tasks/ios/vision/core/sources/MPPVisionTaskRunner.mm

```diff
@@ -29,9 +28,10 @@ @interface MPPVisionTaskRunner () {
 
 @implementation MPPVisionTaskRunner
 
-- (nullable instancetype)initWithCalculatorGraphConfig:(CalculatorGraphConfig)graphConfig
+- (nullable instancetype)initWithCalculatorGraphConfig:(mediapipe::CalculatorGraphConfig)graphConfig
                                            runningMode:(MPPRunningMode)runningMode
-                                       packetsCallback:(PacketsCallback)packetsCallback
+                                       packetsCallback:
+                                           (mediapipe::tasks::core::PacketsCallback)packetsCallback
```

Comment:
Any reason these lines were changed (this and the one above)?  I assume this was just part of a merge.

Reply:
I think so. I use the clang formatter to format files. The one during merge does not format exactly the same way. So any inconsistencies created by clang formatter should get resolved during merge.